app/src/behaviors/behavior_rgb_underglow.c

```diff
@@ -18,11 +18,35 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 static int behavior_rgb_underglow_init(const struct device *dev) { return 0; }
 
+static int on_keymap_binding_to_absolute(struct zmk_behavior_binding *binding,
```

Comment:
I'm not sure about this name, maybe something like "preprocess on central"?

Reply:
That feels like naming it after the current one consumer of the API, versus declaring the generic intent of the API.

Reply:
I feel like the "'on_keymap_binding_to_absolute" actually names it after the current one consumer? "preprocess" is much more general then "to absolute", and this callback is only executed on the central so I think it's worth mentioning.

Reply:
Hmm..... `binding_convert_central_state_dependent_params`?